pkgs/data/fonts/dancing-script/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
+  pname = "dancing-script";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "impallari";
+    repo = "DancingScript";
+    rev = "f7f54bc1b8836601dae8696666bfacd306f77e34";
+    sha256 = "B9oAZFPH3dG/Nt5FfKfFVJYtfUKGK0AXNkQHRC7IgdU=";
+  };
+
+  installPhase = ''
+    install -m444 -Dt $out/share/fonts/truetype fonts/ttf/*.ttf
```

Comment:
If this is just the font we can also only use fetchFromGitHub and postFetch.

Reply:
I was kind of questioning myself whether I should explore this approach. Thanks to your comment, I understand it was not such a bad idea and found an example in the source code as an example.